alignments/sweet-gcmd-mapping.ttl

```diff
@@ -0,0 +1,525 @@
+gcmd:dc18db4d-2184-453e-ba0a-86c83a9bede0
+    skos:exactMatch sorealv:LavaPlain .
+gcmd:cefe2205-809c-4386-915e-a8737ae8e68e
+    skos:exactMatch sorealv:Volcano .
+gcmd:7c2e1960-ae20-46a9-acf1-a3e71542fbb4
+    skos:exactMatch sorealv:Volcano .
+gcmd:58c12630-a889-44c1-a951-56bbbe9758c9
+    skos:exactMatch sorealt:FractureZone .
+gcmd:ca874a66-f3a8-4099-978c-4684944dc348
+    skos:exactMatch sorealt:RiftValley .
+gcmd:0bd4d492-4911-4a6a-afaa-34899a80294b
+    skos:exactMatch sorealt:RiftValley .
+gcmd:46172bbe-8bf0-49a0-848f-129c089aeb8e
+    skos:exactMatch sorealt:TectonicLandform .
+gcmd:2b1f7993-2d54-40de-abc4-3909f619ad4e
+    skos:exactMatch sorealp:Park .
+gcmd:c34ea556-10bd-4665-9f22-68b5d05c9aea
+    skos:exactMatch sorealo:Mountain .
+gcmd:8b7f66ea-d481-4641-9dbf-da90ca3ad9c9
+    skos:exactMatch sorealo:Mountain .
+gcmd:694e18ec-ceaf-4070-9763-f3ee6dbd6b5b
+    skos:exactMatch sorealo:Plateau .
+gcmd:0baf564f-f942-4aeb-9b75-30b838f28f3f
+    skos:exactMatch sorealo:Plateau .
+gcmd:ca091be1-4762-49ec-859b-a1a2fcb8e038
+    skos:exactMatch sorealo:Ridge .
+gcmd:97298feb-6991-4d68-8337-177460e436ad
+    skos:exactMatch sorealo:Ridge .
+gcmd:d37d51e0-1bef-473a-9221-6713166762f9
+    skos:exactMatch sorealg:Arete .
+gcmd:8e73bff6-c2f9-46a6-963b-8ef09dd7f5f3
+    skos:exactMatch sorealg:Arete .
+gcmd:5e012809-98cf-468f-bdf7-7cea8569d3ab
+    skos:exactMatch sorealg:Esker .
+gcmd:158a8764-a6e4-4d28-a1b9-b2ab91e09995
+    skos:exactMatch sorealg:Esker .
+gcmd:3c78951a-0293-4fb0-baff-ec7372fe784d
+    skos:exactMatch sorealg:GlacialLandform .
+gcmd:b895f4b5-5273-49ef-883f-b67d9f199505
+    skos:exactMatch sorealg:GlacialLandform .
+gcmd:4f590d94-110c-4762-9171-aba6d24af6a0
+    skos:exactMatch sorealg:Moraine .
+gcmd:2575cfaf-1a09-48b6-acb9-fda23b6f4719
+    skos:exactMatch sorealg:Moraine .
+gcmd:b1ce822a-139b-4e11-8bbe-453f19501c36
+    skos:exactMatch sorealg:RockGlacier .
+gcmd:ee2af62b-9f76-440c-aa9b-77940468b8f4
+    skos:exactMatch sorealg:RockGlacier .
+gcmd:fcbf8f96-ac53-41b6-9c98-a87425a4ec82
+    skos:exactMatch sorealg:RockGlacier .
+gcmd:2d98cbaf-8c82-46e6-9962-a5e63918fe66
+    skos:exactMatch sorealg:RockGlacier .
+gcmd:bffac466-83aa-4060-a378-6d2d6e49f2a1
+    skos:exactMatch sorealg:TillPlain .
+gcmd:2bea72da-2cf3-403c-adb9-9d963eb71536
+    skos:exactMatch sorealg:TillPlain .
+gcmd:6a426480-c58f-4b6b-8e35-0975b7f6edb5
+    skos:exactMatch soreal:LandSurface .
+gcmd:f36d71c6-f2ad-49c4-809f-09b4f0688412
+    skos:exactMatch soreal:Landscape .
+gcmd:e25ce36c-eacd-447a-9d73-ccc8a7e3a328
+    skos:exactMatch sorealf:Canyon .
+gcmd:a78c946a-9529-4643-b002-1aa2ac9cfed6
+    skos:exactMatch sorealf:Canyon .
+gcmd:272700c5-d762-452b-8e9f-130e3a51efb5
+    skos:exactMatch sorealf:DrainageBasin .
+gcmd:d71f94cb-e773-487a-a8ff-9c5f11c1dbc4
+    skos:exactMatch sorealf:FloodPlain .
+gcmd:ba37314d-ec38-4e67-bf30-7e1fdc6bfbad
+    skos:exactMatch sorealf:FloodPlain .
+gcmd:cb5193ab-2d7a-4b35-b7ec-f16ce78ae270
```

Comment:
Are these all `exactMatch`? For example I see two separate exact matches to FluvialLandform and to FloodPlain here. That doesn't seem right. Or is it because GCMD manages its polyhierarchy with separate URIs for the same concept on different branches?

Reply:
Correct.  I considered them to be `exactMatch` match based on the GCMD term and definition, using SWEET's _realm_ as a context indicator.  

There are two matches from [SWEET:FluvialLandform](http://sweetontology.net/realmLandFluvial/FluvialLandform) to GCMD _fluvial landform_ because GCMD has two unique concept URIs which, in this case, have the exact same label and definition --- see [[1](https://gcmd.earthdata.nasa.gov/kms/concept/cb5193ab-2d7a-4b35-b7ec-f16ce78ae270)] and [[2](https://gcmd.earthdata.nasa.gov/kms/concept/bdc0bd86-a3a3-48fa-b1fb-4ca5d13d4dde)].

This is the same with [SWEET:FloodPlain](http://sweetontology.net/realmLandFluvial/FloodPlain), [SWEET:Gully](http://sweetontology.net/realmLandFluvial/Gully) and a host of others, although not all definitions match exactly word for word as these do.